lib/help.js

```diff
@@ -1,67 +1,149 @@
-const path = require('path');
-const fs = require('fs');
 const config = require('config');
+const Table = require('table-layout');
+const usage = require('command-line-usage');
+const commands = require('./commands');
 
-function pad(text) {
-  return text.padEnd(15);
+function styleCommand(command) {
+  return `${'$'.grey} ${config.cli} ${command.name}`;
 }
 
-exports.run = function() {
-  console.log('');
-  console.log(`Usage: ${config.cli} <command> [arguments]`);
-  const files = fs
-    .readdirSync(__dirname)
-    .map(file => {
-      const stats = fs.statSync(path.join(__dirname, file));
-      if (stats.isDirectory()) {
-        return fs.readdirSync(path.join(__dirname, file)).map(f => path.join(file, f));
+const owlbert = () => {
+  // http://asciiart.club
+  return `          ░                                        ░
+          ░░                                       ░░
+        ░░░░░                                     ░░░░ ░
+       ░░░░░░░                                    ░░░░░░
+       ▐░░░░░░▄                                 ░░░░░░░░░
+       ░░░░░░░░░▐░░░                        ░░░░░░░░░░░░░   ${config.cli.blue.bold}
+        ░░░░░░░░░░░░░░░░░▐▐▐░░░░░░░░░░▐▐░░░░░░░░░░░░░░░░░░
+      ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░    ${
+        `a utlity for interacting with ReadMe.io`.bold
```

Comment:
We could probably drop the .io in _ReadMe.io_? @mjcuva could maybe confirm or deny if this is the direction we're heading in with the new homepage domain and all.

Reply:
Yes! We are in the bold new era of being a .com and not needing that pesky tld everywhere

Reply:
I'll change this to just be "a utlity for interacting with ReadMe", and also fix the typo in "utility" while I'm at it.